syntax = "proto3";

package backup;

// Serviço de Backup
service BackupService {
  rpc LogTransaction (TransactionRequest) returns (TransactionResponse) {}
}

// Mensagem de requisição para LogTransaction
message TransactionRequest {
  string terminal_id = 1;
  string vehicle_class = 2;
  string vehicle_name = 3;
  string status = 4;
}

// Mensagem de resposta para LogTransaction
message TransactionResponse {
  bool success = 1;
  string message = 2;
}